src/components/DashStat.container.js

```diff
@@ -27,8 +27,10 @@ export const DashStatTotalTrees = compose(
   connect(
     (state) => ({
       data:
-        state.trees.treeCount !== null
-          ? countToLocaleString(state.trees.treeCount)
+        state.verify.treeCount + state.verify.verifiedTreeCount !== null
```

Comment:
This null check doesn't work – the `+` operator always returns a valid number by interpreting `null` as `0`.
You can probably just remove this check.